### ISSUE TITLE
Consider publishing a list of apt & pip packages for the colab docker image

### ISSUE THREAD

**Issue by craigcitro** [googlecolab/colabtools#1002]
As in https://discourse.jupyter.org/t/a-new-open-source-jupyter-blogging-system-hosted-on-github-pages/3334/5, folks are looking to be able to easily execute notebooks across both binder and colab. @betatim asked about this on twitter https://twitter.com/betatim/status/1229460317933113344. 

Publishing our docker image would be complex, but I believe we should be able to push out the list of installed packages in our container to github, which would be a good start; @jakevdp and I should chat and see if I'm tricking myself with that claim. :) A secondary benefit here would be that users could easily answer "hey, what version of <thispkg> was installed last week when I ran this notebook?"

**Comment by tnovikoff** [comment]
In case anybody comes across this thread who doesn't know how to list the packages installed in a given Colab VM, `!pip list` will list out all the installed packages at any time.

**Comment by francesco-ballarin** [comment]
Hi,
a year later, is there any update on this?

Since the python 3.7 upgrade on Colab we have to manually package some libraries (manually = open Colab on a browser, build a wheel, download the wheel, upload it somewhere else), and it would be great to have a docker image of the Colab environment to make this process automatic, at least until #1880 is fixed.

Thanks

**Comment by tcapelle** [comment]
I would also love to have this, I want to be able to test that our notebooks keep colab compatibility.

**Comment by francesco-ballarin** [comment]
@craigcitro Should https://github.com/googlecolab/backend-info close this issue?

**Comment by craigcitro** [comment]
@francesco-ballarin yes! at least, that repo is a first step in the right direction, but this one is worth closing (and we can file more specific issues in the future).
